backend/src/appointment/database/repo/subscriber.py

```diff
@@ -71,8 +71,24 @@ def update(db: Session, data: schemas.SubscriberIn, subscriber_id: int):
     return db_subscriber
 
 
+def disable(db: Session, subscriber: models.Subscriber):
+    """Disable a given subscriber"""
+    subscriber.active = False
```

Comment:
You'll want to set the minimum_iat_time to now as well. This will cause any jwt tokens with an iat (issued at) of less than the minimum_iat_time to be invalidated. Thus logging them out.

Reply:
Perfect, exactly what I was looking for, thanks!